refinery/config/settings/base.py

```diff
@@ -637,3 +637,6 @@ def get_setting(name, settings=local_settings, default=None):
 }
 REFINERY_VISUALIZATION_REGISTRY = \
     "https://github.com/refinery-platform/visualization-tools/"
+
+# The upper bound of documents returned from a Solr query
+REFINERY_SOLR_DOC_LIMIT = 10000000
```

Comment:
If this is not user-configurable then shouldn't this be just a data_set_manager constant?